src/qt/optionsdialog.cpp

```diff
@@ -295,10 +295,22 @@ void OptionsDialog::on_resetButton_clicked()
 
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
-    /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    QMessageBox config_msgbox;
+    config_msgbox.setIcon(QMessageBox::Information);
+    //: Window title text of pop-up box that allows you to open up your configuration file.
+    config_msgbox.setWindowTitle(tr("Configuration options"));
+    /*: Explanatory text about the priority order of instructions considered by client.
+        The order from high to low being: command-line, configuration file, GUI settings. */
+    config_msgbox.setText(tr("The configuration file is used to specify advanced user options which override GUI settings. "
+                             "Additionally, any command-line options will override this configuration file."));
+
+    QPushButton* open_button = config_msgbox.addButton(tr("Continue"), QMessageBox::ActionRole);
```

Comment:
Not sure if `QMessageBox::ActionRole` is a good choice.

[`QMessageBox::ActionRole`](https://doc.qt.io/qt-5/qmessagebox.html#ButtonRole-enum): 
> Clicking the button causes _changes to the elements within the dialog_.

Maybe `QMessageBox::AcceptRole` or `QMessageBox::YesRole`?

In case of `QMessageBox::AcceptRole` the button positions are evaluated according to the system-dependant styles.
And s/`open_button->setDefault(true);`/`config_msgbox.setDefaultButton(open_button);`/ is required.

Reply:
Both `QMessageBox::AcceptRole` and `QMessageBox::YesRole` are leading to changing the ordering of `continue` and `cancel` buttons. 
I am trying to keep ordering buttons in the `Configuration Options` window and `Options window` same to maintain consistency but both `AcceptRole` and `YesRole` mess that up.
![Screenshot from 2021-09-10 18-47-46](https://user-images.githubusercontent.com/85434418/132860071-f00bcc45-87b1-4ff6-b079-6d6a794a71f7.png)
I guess there must be some way of using one of these roles without disrupting the ordering, but I cannot find it yet.